Sources/ComposableArchitecture/Store.swift

```diff
@@ -427,8 +427,8 @@ public final class Store<State, Action> {
   @_spi(Internals)
   public
     func scope<ChildState, ChildAction>(
-      state: ToState<State, ChildState>,
       id: ScopeID<State, Action>?,
+      state: ToState<State, ChildState>,
```

Comment:
Reordering `id` to be first while I'm in here so that `state` and `action` remain grouped together.